source/process/training.rst

```diff
@@ -294,20 +294,37 @@ An easy way to check in with team members about how things are going.
 
 Use these one-on-one or in a group as a way to open conversations about what to keep and what to change in how we do things.
 
+Shoulder Check 
+---------------------------------------------
+
+When a new owner takes over a process or a project from a previous owner, there are a finite number of "blindspots" of which the original owner is aware and the new owner will need to understand. 
+
+Using the analogy of changing lanes while driving a vehicle and learning to do a "shoulder check" for information that is not visible from standard controls, we have a process for the new owner and previous owner to jointly review processes until the transfer is complete. 
+
+This process is similar to `Mini-boss, End-boss <https://docs.mattermost.com/process/training.html?highlight=mini%20boss#mini-boss-end-boss>`_, except that the mini-boss is also the new owner of a process, and not only a reviewer. Shoulder checks should be requested by new owners to avoid "crashing":
+
+ - Making changes to systems that break existing processes and may lose data and hurt the productivity of others downstream without notice and without a replacement system in place (behavior known as `"Dead Tarzan" <https://docs.mattermost.com/process/training.html?highlight=dead%20tarzan#dead-tarzan>`_). 
+ - Repeatedly investing in mis-prioritized projects due to a misunderstanding of requirements from project stakeholders and insufficient confirmation of intended outcomes. 
+
+Even when not crashing, as part of our `Self Awareness value <https://docs.mattermost.com/process/handbook.html?highlight=self%20awareness#values>`_, top team members will constantly be seeking feedback and review from people around the company. 
+
 Mini-boss, End-boss
 ---------------------------------------------
 
-When reviewing user interface design, pull requests, or marketing materials, there are ideally two reviewers:
+After completing the initial draft of a project, there may often be more than one reviewer to approve changes. This may be for different disciplines to review the work (for example, both development and design teams reviewing code changes to the user experience) and it may also be for reviewers with different levels of experience to share feedback. 
+
+When reviewing significant user interface changes, code changes, responses to community or customers, or changes to systems or marketing material changes, it is ideal to have at least two reviewers:
 
-- **Mini-boss**: Reviewer with less experience to do the first review
-- **End-boss**: More experienced reviewer to do the final review
+- **Mini-boss**: Reviewer less experienced in domain or Mattermost standards for the first review
+- **End-boss**: Reviewer less experienced in domain or Mattermost standards for the final review for the discipline (e.g. development, design, documentation, etc.) 
```

Comment:
Is this supposed to be "more experienced"? 

I'm not sure I understand the sentence if it's "less experienced"